sentry.gradle

```diff
@@ -92,7 +92,10 @@ gradle.projectsEvaluated {
               } catch (FileNotFoundException e) {
                   project.logger.info("file not found '$propertiesFile' for '$variant'")
               }
-              def cliExecutable = sentryProps.get("cli.executable", "$reactRoot/node_modules/@sentry/cli/bin/sentry-cli")
+
+              def cliExecutable = sentryProps.get(
+                  "cli.executable",
+                  new File(["node", "--print", "require.resolve('@sentry/cli/package.json')"].execute(null, rootDir).text.trim()).getParentFile().getAbsolutePath() + "/bin/sentry-cli")
```

Comment:
can we instead read the file in advance and if something goes wrong, fallback to the hardcoded path -> `"$reactRoot/node_modules/@sentry/cli/bin/sentry-cli"`? as before?

eg
// read from @sentry/cli/package.json
// read from cli.executable
// use it either, `cli.executable` first, then the one from `@sentry/cli/package.json` and if none of them have a valid path, `$reactRoot/node_modules/@sentry/cli/bin/sentry-cli`.